app/src/main/java/com/x8bit/bitwarden/data/auth/repository/util/AuthDiskSourceExtensions.kt

```diff
@@ -100,6 +101,47 @@ val AuthDiskSource.userAccountTokensFlow: Flow<List<UserAccountTokens>>
         }
         .distinctUntilChanged()
 
+/**
+ * Returns the current list of [UserKeyConnectorState].
+ */
+val AuthDiskSource.userIsUsingKeyConnectorsList: List<UserKeyConnectorState>
```

Comment:
⛏️ Should we rename these too, sorry for not calling this out with the other comment. I am not sure if I feel like this is too much of a nit.

Reply:
Nope, that's a good change and I am on it!